Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -205,6 +206,7 @@ def bottle_formula(f)
         tab.poured_from_bottle = false
         tab.HEAD = nil
         tab.time = nil
+        tab.changed_files = changed_files
```

Comment:
My only concern with putting it in the tab is that this may be a very long list. How many values are there for e.g. `mysql`?